src/builder/Actions.sol

```diff
@@ -329,6 +367,15 @@ library Actions {
         address token;
     }
 
+    struct MorphoVaultSupplyContext {
```

Comment:
Why is there both a `MorphoVaultSupplyActionContext` and a `MorphoVaultSupplyContext`? Both seem to be used too

Reply:
oops